src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/WebJobs.Logging.ApplicationInsights.csproj

```diff
@@ -50,7 +54,7 @@
     without all of the others in this solution. If changes to WebJobs.Host are needed here, re-introduce the
     project reference. 
     -->
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.33" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.39" />
```

Comment:
This was referencing a really old version, so I'm bumping this to the latest released.